### DISCUSSION TITLE
Changing baseBranches for one task

### DISCUSSION THREAD

**Discussion by Marvin0098** [renovatebot/renovate#23506] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

AWS CodeCommit

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

My current renovate config looks like this:

```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "platform": "codecommit",
  "endpoint": "https://git-codecommit.eu-central-1.amazonaws.com/",
  "enabled": true,
  "baseBranches": ["develop"],
   .....
}
```

I want to use the `osvVulnerabilityAlerts` task but I want to check against all my upstram branches (develop, qa, master).
We just want renovate to check against the develop branch for new dependency versions.

Is there a way to specify a different set of baseBranches for just one task (`osvVulnerabilityAlerts` in this case)

```
{
  "osvVulnerabilityAlerts": true
}
```

### What you tried so far.

Using baseBranches in a packageRule didnt work

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by viceice** [comment, marked as answer]
you need to set a never occurred schedule by default and disable it for your develop branch.
hat way you can enable vulnerability updates which also override schedule.

search discussions for security only updates for sample

**Reply by Marvin0098** [reply]
I have checked previous discussions but have not found anything in regards to schedules. Was only able to find this discussion:
https://github.com/renovatebot/renovate/discussions/15490#discussioncomment-5541923
But that did not work on my end. Can you please give provide some example code?

**Reply by viceice** [reply]
why does that solution doesn't work for you? you need to add a new rule to enable `develop` base branch updates again after the disable rule
